api/volume_server.go

```diff
@@ -379,6 +380,38 @@ func (vs *VolumeServer) SetProperty(w http.ResponseWriter, req *http.Request) {
 	w.WriteHeader(http.StatusNoContent)
 }
 
+func (vs *VolumeServer) GetPrivileged(w http.ResponseWriter, req *http.Request) {
```

Comment:
Had to add this endpoint in to check what privileges the parent volume had so we can apply the same to the COW. We can't pass it in as a flag to the plugin binary as the plugins flags are statically configured during garden startup.